### DISCUSSION TITLE
Packaging recommendations?

### DISCUSSION THREAD

**Discussion by melMass** [noib3/nvim-oxi#150] [Q&A] [answered]
Hi,

Thanks for these bindings! I made a POC [here](https://github.com/melMass/echo.nvim) and managed to automate the build / [release](https://github.com/melMass/echo.nvim/releases/tag/latest).

What would be a good approach to ship these now? I think the lazy plugin spec has a `build` field (I am about to test it out more), but maybe gathering good practice for popular PM would be nice.

**Comment by noib3** [comment, marked as answer]
I think having the `build` script fetch the compiled binaries from the releases page is the easiest way.

**Reply by melMass** [reply]
I since did, but did not find how to test it properly.
In case it is useful to others [build.lua](https://github.com/melMass/echo.nvim/blob/main/build.lua)
